cmd/aro/rp.go

```diff
@@ -52,6 +52,7 @@ func rp(ctx context.Context, log, audit *logrus.Entry) error {
 	if _env.IsLocalDevelopmentMode() {
```

Comment:
Is it needed in development?

Reply:
Yes it is needed as there will a need to create a bloburl which will contain the storage account name.